design/concise-tidy-design.Rmd

```diff
@@ -90,12 +133,11 @@ setMethod(
   f = "tidy",
   signature = signature(x = "CohortSizeParts"),
   definition = function(x, ...) {
-    rv <- tibble::tibble(
+    tibble::tibble(
       part = seq_along(x@cohort_sizes),
       cohort_size = x@cohort_sizes
-    )
-    class(rv) <- c(paste0("tbl_", class(x)), class(rv))
-    return(rv)
+    ) %>%
```

Comment:
I would suggest to use new `|>` consistently and adjust minimum R version in the package accordingly to 4.2.1

Reply:
that said, in package code we should generally avoid the pipe, and default to several assignments of variables. (this simplifies debugging and can add additional readability through meaningful variable names)